openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2688,6 +2688,28 @@ in the manner described in [@RFC6838].
 * Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Provisional registration? No
 
+### application/keyattestation+jwt
+
+* Type name: `application`
+* Subtype name: `keyattestation+jwt`
```

Comment:
```suggestion
* Subtype name: `key-attestation+jwt`
```

Reply:
would need to update the actual text and example(s) in the document too